internal/config/config_test.go

```diff
@@ -0,0 +1,260 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package config
+
+import (
+	"bytes"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestRead(t *testing.T) {
+	require := require.New(t)
+
+	cacheLen := 2
+	require.Equal(cacheLen, len(cache.values)) // from the init() of log.go
+	envSetup()                                 // clears the cache
```

Comment:
It would be best to use the suite pkg to avoid manually calling this setup. You could use the same setup for all the tests.